README.md

```
@@ -1 +1,224 @@
+## Table of Contents
+
+- [Contributors](#contributors)
+- [Built With](#built-with)
+- [Getting Started](#getting-started)
+- [Features](#features)
+- [Technical Challenges](#technical-challenges)
+- [Wiki](#wiki)
+
+## Contributors
+
+| [<img src="https://github.com/pintordev.png" width="100">](https://github.com/pintordev)<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[김호현](https://github.com/pintordev) 🚩 | [<img src="https://github.com/habin08090.png" width="100">](https://github.com/habin08090)<br>[김하빈](https://github.com/habin08090) | [<img src="https://github.com/zrp0x0.png" width="100">](https://github.com/zrp0x0)<br>[신홍규](https://github.com/zrp0x0) | [<img src="https://github.com/Ksinny.png" width="100">](https://github.com/Ksinny)<br>[이경신](https://github.com/Ksinny) |
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)